docs/tutorials/studio-tutorial.md

```diff
@@ -572,6 +572,15 @@ This panel lets you define tests for the services you write by specifying test d
 
 ## Create a Service Test 
 
+### Create connections test data
+Click the `generate` button to automatically generate test data if possible. Otherwise, please manually create test data.
+#### Data Type -- Relational
+Add relational data tables based on databases to which the service's mapping connects. It's unnecessary to provide all columns with test data. 
+The columns used in the service, however, require test data. Besides, the primary keys have to be distinct.
```

Comment:
Lets add something like: 
Test data for relational is defined through csv values for each table. You must specify the schema name, table name and the corresponding csv alues.